Makefile

```diff
@@ -27,13 +27,5 @@ clean-test:
 	rm -f .coverage
 	rm -fr htmlcov/
 
-test:
-	python setup.py test --pytest-args="--cov=./djmoney tests"
```

Comment:
Make targets are a nice way of consolidating the entrypoints to a code base. Either as a means of quickly running commands or as a file that you read straight after the README to see how things are structured :)